pkg/types/azure/defaults/machines.go

```diff
@@ -0,0 +1,20 @@
+package defaults
+
+import (
+	"fmt"
+)
+
+func BootstrapMachineType(region string) string {
+	instanceClass := InstanceClass(region)
+	return fmt.Sprintf("%s_D4s_v3", instanceClass)
+}
+
+func MasterMachineType(region string) string {
+	instanceClass := InstanceClass(region)
+	return fmt.Sprintf("%s_D4s_v3", instanceClass)
```

Comment:
this should be `_DS4_v2`

Reply:
As discussed offline, the correct constants are "Standard_D2s_v3" and "Standard_D4s_v3".